api/src/main/java/com/messagebird/objects/VoiceMessage.java

```diff
@@ -171,6 +172,22 @@ public void setIfMachine(IfMachineType ifMachine) {
         this.ifMachine = ifMachine;
     }
 
+    /**
+     * The time (in milliseconds) to analyze if a machine has picked up the phone.
+     * Used in combination with the delay and hangup values of the ifMachine attribute.
+     * Minimum: 400, maximum: 10000. Default: 7000
+     * @return value of machine timeout
+     */
+    public int getMachineTimeout() { return machineTimeout; }
+
+    /**
+     * The time (in milliseconds) to analyze if a machine has picked up the phone.
+     * Used in combination with the delay and hangup values of the ifMachine attribute.
+     * Minimum: 400, maximum: 10000. Default: 7000
+     * @param machineTimeout value of machine timeout
+     */
+    public void setMachineTimeout(int machineTimeout) { this.machineTimeout = machineTimeout; }
```

Comment:
Check if the user is adding a write machineTimeout value here(min 400 max 10000), if not throw an error